src/Microsoft.PowerShell.Commands.Utility/commands/utility/convert-HTML.cs

```diff
@@ -22,7 +22,7 @@ namespace Microsoft.PowerShell.Commands
     /// </summary>
 
     [Cmdlet(VerbsData.ConvertTo, "Html", DefaultParameterSetName = "Page",
-        HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113290", RemotingCapability = RemotingCapability.None)]
+        HelpUri = "http://go.microsoft.com/fwlink/?LinkID=113290", RemotingCapability = RemotingCapability.None)]
```

Comment:
Could you revert this to https please. TLS connections should be the default for URLs.